crates/rerun/src/run.rs

```diff
@@ -539,3 +548,35 @@ pub fn setup_ctrl_c_handler() -> (tokio::sync::broadcast::Receiver<()>, Arc<Atom
     .expect("Error setting Ctrl-C handler");
     (receiver, shutdown_return)
 }
+
+// ----------------------------------------------------------------------------
+
+use re_log::external::log;
+
+struct StrictLogger {}
+
+impl log::Log for StrictLogger {
+    fn enabled(&self, metadata: &log::Metadata<'_>) -> bool {
+        match metadata.level() {
+            log::Level::Error | log::Level::Warn => true,
+            log::Level::Info | log::Level::Debug | log::Level::Trace => false,
+        }
+    }
+
+    fn log(&self, record: &log::Record<'_>) {
+        let level = match record.level() {
+            log::Level::Error => "error",
+            log::Level::Warn => "warning",
+            log::Level::Info | log::Level::Debug | log::Level::Trace => return,
+        };
+
+        eprintln!("{level} logged in --strict mode: {}", record.args());
+        eprintln!(
+            "{}",
+            crate::crash_handler::callstack_from(&["log::__private_api_log"])
```

Comment:
where does this string come from? Does it need to be updated if logging crate is updated?

Reply:
It comes from the callstack, and yes, it will need to.

The full callstack leading up to `StructLogger` is:

```
   1: rerun::crash_handler::callstack_from
             at rerun/src/crash_handler.rs:229:21
   2: <rerun::run::StrictLogger as log::Log>::log
             at rerun/src/run.rs:574:25
   3: <&T as log::Log>::log
             at log-0.4.17/src/lib.rs:1318:9
      <re_log::multi_logger::MultiLogger as log::Log>::log
             at re_log/src/multi_logger.rs:60:13
   4: log::__private_api_log
             at log-0.4.17/src/lib.rs:1597:5
```

and I want to trim away as much irrelevant stuff as possible